Fix 186 (#189)
ait-energy/en.ergie.at: content/projekte/INTERSTORES/index.md

---
title: "INTERSTORES"
topics: ["Thermische Energiespeicherung", "Erneubare Energien", "Optimierung"]
authors: [plosieva]
---

Die EU strebt eine rasche Dekarbonisierung des Energiesektors durch das Fit for 55-Paket, den RePowerEU-Plan und den Green Deal an. Großspeichertechnologien zur saisonalen thermischen Energiespeicherung (sTES) (z. B. Becken, Kavernen) können die Bemühungen unterstützen, indem sie die Nutzung fluktuierender Wärmequellen verbessern. INTERSTORES wird sich diesen Herausforderungen stellen und die Dekarbonisierung durch die Anwendung fortschrittlicher sTES-Technologien beschleunigen.

<!--more-->

> Das übergeordnete Ziel von INTERSTORES ist die Demonstration und Umsetzung zweier innovativer sTES-Lösungen (Wiederverwendung alter Becken, unterirdische Kavernen) und die Validierung der Nutzung als Mehrzweck-sTES an den Demostandorten. INTERSTORES zielt darauf ab, die Nachhaltigkeit zu verbessern, indem kosteneffiziente, recycelte Isoliermaterialien verwendet werden, der Einsatz erneuerbarer Energien erhöht wird und herkömmlicher Spritzbeton ersetzt wird. Der Haupteffekt wird in der Senkung der Investitionskosten, des Energiebedarfs, der Verringerung der CO2-Emissionen und der Umwelt- und Bodenbelastung gesehen.

Reviewer: ```suggestion
Das übergeordnete Ziel von INTERSTORES ist die Demonstration und Umsetzung zweier innovativer sTES-Lösungen (Wiederverwendung alter Becken, unterirdische Kavernen) und die Validierung der Nutzung als Mehrzweck-sTES an den Demostandorten. INTERSTORES zielt darauf ab, die Nachhaltigkeit zu verbessern, indem kosteneffiziente, recycelte Isoliermaterialien verwendet werden, der Einsatz erneuerbarer Energien erhöht wird und herkömmlicher Spritzbeton ersetzt wird. Der Haupteffekt wird in der Senkung der Investitionskosten, des Energiebedarfs, der Verringerung der CO<sub>2</sub>-Emissionen und der Umwelt- und Bodenbelastung gesehen.
```

## Factsheet

|   |   |
|---|---|
| **Kurzform** | <mark>INTERSTORES</mark> |
| **Titel** | Internationales Innovationsnetzwerk zur Entwicklung von Kosten- und umweltfreundliche saisonale thermische Energiespeicher |
| **Laufzeit** | 01.01.2024 -- 31.12.2027 |
| **Partner** | 14 |
| **Projektform** | Forschungs- und Innovationsprojekt  |
| **Projektleitung AIT-seitig** | [Abdulrahman Dahash](abdulrahman.dahash(at)ait.ac.at) |

> | **Projektleitung IES-seitig** | [Vikroriia Illyes](viktoria.illyes@ait.ac.at) |

Reviewer: ```suggestion
| **Projekt Koordination** | [Prof. Dr. Peter Bayer](peter.bayer@geo.uni-halle.de) ([MLU](https://www.uni-halle.de/)] |
| **Projektleitung AIT** | [Dr. Abdulrahman Dahash](abdulrahman.dahash(at)ait.ac.at) & [Dr. Viktoria Illyés](viktoria.illyes@ait.ac.at) |
```

4. **Markt**

    Demonstration von zwei marktkonkurrierbaren, reproduzierbaren saisonalen thermischen Energiespeicherlösungen.

### Projektpartner

- [MARTIN-LUTHER-UNIVERSITAT HALLE-WITTENBERG](https://geo.uni-halle.de/appliedgeo/home/) (Projektkoordinator)
- [AIT Austrian Institute of Technology GmbH](https://www.ait.ac.at/ueber-das-ait/center/center-for-energy)
- [E.ON UK Infrastructure Services Limited](https://www.eonenergy.com/)
- [Stichting Global Energiesprong Alliance](https://www.energiesprong.org/)
- [GREENFLEX](https://geo.uni-halle.de/appliedgeo/home/)
- [GEOLOGIAN TUTKIMUSKESKUS](https://www.gtk.fi/en/)
- [FACHHOCHSCHULE ZENTRALSCHWEIZ - HOCHSCHULE LUCH](https://www.hslu.ch/de-ch/)
- [IN-Campus GmbH](https://incampus.de/)
- [INOSENS DOO NOVI SAD](https://inosens.rs/)
- [Planungsgruppe M+M AG](https://www.pgmm.com/de/home)
- [TECHNISCHE HOCHSCHULE INGOLSTADT](https://energieinstitut-linz.athttps://www.thi.de/forschung/institut-fuer-neue-energie-systeme-ines/)
- [VANTAAN ENERGIA OY](https://www.vantaanenergia.fi/en/)
- [TEKNOLOGIAN TUTKIMUSKESKUS VTT OY](https://www.vttresearch.com/en)
- [WIZ DEVELOPMENT & SERVICES SRL](https://wizresearch.com/)

> 

Reviewer: ```suggestion

### Weitere Informationen

- Projekthomepage: [interstores.eu](https://interstores.eu/)
- AIT Projektinformationen: [www.ait.ac.at](https://www.ait.ac.at/themen/waermenetze-und-kaeltenetze-fuer-stadt-und-industrie/projekte/saisonalen-thermische-energiespeicher)

```
